qiskit_nature/algorithms/ground_state_solvers/adapt_vqe.py

```diff
@@ -40,15 +42,34 @@
 
 
 class AdaptVQE(GroundStateEigensolver):
-    """A ground state calculation employing the AdaptVQE algorithm."""
-
+    """A ground state calculation employing the AdaptVQE algorithm.
+
+    The performance of AdaptVQE can significantly depend on the choice of gradient method, QFI
+    solver (if applicable) and the epsilon value.
+
+    To reproduce the default behavior of AdaptVQE prior to Qiskit Nature 0.4 you should supply
+    `delta=1` explicitly. This will use a finite difference scheme for the gradient evaluation
+    whereas after version 0.4 a parameter shift gradient will be used.
+    [https://qiskit.org/documentation/tutorials/operators/02_gradients_framework.html]
```

Comment:
Let's be a bit more explicit here:
```suggestion
    For more information refer to the gradient framework of Qiskit Terra:
    https://qiskit.org/documentation/tutorials/operators/02_gradients_framework.html
```